librarian_packages/hera_librarian/tests/test_base_store.py

```diff
@@ -0,0 +1,183 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2019 the HERA Collaboration
+# Licensed under the 2-clause BSD License
+
+"""Test code in hera_librarian/store.py
+
+"""
+
+from __future__ import print_function, division, absolute_import
+import pytest
+import os
+import json
+from hera_librarian import base_store, RPCError
+
+from . import ALL_FILES, filetypes, obsids, md5sums, pathsizes
+
+
+@pytest.fixture
+def local_store():
+    return base_store.BaseStore("local_store", "/tmp", "localhost")
```

Comment:
To be really paranoid it might be best to have the store's temporary directory created dynamically with `tempfile.mkdtemp`, since otherwise repeated or simultaneous invocations of this test suite might stomp on each other in the shared `/tmp` directory. You'd have to parametrize all of the `/tmp` instances below into a variable, which would be pretty annoying, though.

Reply:
This is a good point. I've gone through and done this.